CHANGELOG.md

```diff
@@ -1,5 +1,23 @@
 # Braintree iOS SDK Release Notes
 
+## unreleased
+* BraintreePayPalNativeCheckout
+  * Rename `BTPayPalNativeError` to `BTPayPalNativeCheckoutError`
+  * Add `BTPayPalNativeCheckoutError.deallocated` = 11
+* BraintreeAmericanExpress
+  * Add `BTAmericanExpressError.deallocated` = 2
+* BraintreeCore
+  * Add `BTAPIClientError.deallocated` = 2
+  * Add `BTHTTPError.deallocated` = 12
```

Comment:
Hello, can you tell me how the clients get deallocated during the session? Was this observed in specific modules, cases?